traefik/templates/rbac/clusterrole.yaml

```diff
@@ -47,4 +47,14 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - extensions
+      - networking.k8s.io
+    resources:
+      - ingresses
+      - ingressclasses
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
This creates duplication. The addition should just be to the existing apigroups

Reply:
`extensions` -> `ingresses` -> `get`, `watch`, `list` appear to be covered already, however, `networking.k8s.io` is not, and could be added to the existing definition at `apiGroups[1]`